.github/workflows/ci.yml

```diff
@@ -5,24 +5,98 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Set CACHE_DATE
-      run: echo "CACHE_DATE=$(date --utc +%Y%m%dT%H%M%SZ)" >> $GITHUB_ENV
-    - run: ./devel/pull
-    - run: ./devel/build
-    - if: ${{ github.event_name != 'pull_request' }}
+
+    - uses: docker/setup-buildx-action@v1
+      id: builder-image-builder
+
+    - uses: docker/setup-buildx-action@v1
+      id: base-image-builder
+
+    - name: Show builder-image-builder name
+      run: echo ${{ steps.builder-image-builder.outputs.name }}
+
+    - name: Show base-image-builder name
+      run: echo ${{ steps.base-image-builder.outputs.name }}
+
+    # Set CACHE_DATE in your environment to force layers after our custom cache
+    # point to be re-built. See the ARG CACHE_DATE line in the Dockerfile for more
+    # information.
+    - name: Set CACHE_DATE and GIT_REVISION env vars
+      run: |
+        echo "CACHE_DATE=$(date --utc +%Y%m%dT%H%M%SZ)" >> $GITHUB_ENV
+        echo "GIT_REVISION=$(git describe --tags --abbrev=40 --always --dirty || true)" >> $GITHUB_ENV
```

Comment:
I like to do `| tee -a` instead of `>>` so that the values are visible in the workflow logs.